package.json

```diff
@@ -67,8 +67,27 @@
 	},
 	"devDependencies": {
 		"ava": "^2.3.0",
+		"execa_test_double": "^4.0.0",
+		"mockery": "^2.1.0",
 		"proxyquire": "^2.1.0",
 		"sinon": "^8.0.1",
 		"xo": "^0.25.3"
+	},
+        "xo": {
+		"rules": {
+			"ava/no-import-test-files": [
+				"off",
+				{
+					"files": [
+						"test/*"
+					]
+				}
+			]
+		}
```

Comment:
Why is this needed?

Reply:
It's not really needed with the current xo version.